tests/test-notebooks/test_notebooks.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import glob
+import nbformat
+import os
+import sys
+import time
+import traceback
+import unittest
+
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbconvert.preprocessors.execute import CellExecutionError
+
+
+class test_notebooks(unittest.TestCase):
+
+  def _exec_notebook(self, ep, notebook_filename, nbpath):
+    with open(notebook_filename) as f:
+      nb = nbformat.read(f, as_version=nbformat.current_nbformat)
+      try:
+        out = ep.preprocess(nb, {'metadata': {'path': nbpath}})
+      except CellExecutionError:
+        print('-' * 60)
+        traceback.print_exc(file=sys.stdout)
+        print('-' * 60)
+        self.assertTrue(False,
+                        'Error executing the notebook %s. See above for error.'
+                        % notebook_filename)
+
+  def test_all_notebooks(self):
+    """ Test all notebooks except blacklist. """
+    blacklist = ['gan.ipynb']
+    pythonkernel = 'python' + str(sys.version_info[0])
+    nbpath = '../../docs/notebooks/'
```

Comment:
Do you know how this nbpath works? My impression was that travis calls pytest within the repo's director and not within `tests/test-notebooks/`, which is what assume this nbpath is referring to.